# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wmacpi/prefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- wmacpi-2.1_orig/Makefile	2005-01-05 07:02:08.000000000 +0100
+++ wmacpi-2.1/Makefile	2005-06-18 06:09:45.000000000 +0200
@@ -26,7 +26,7 @@
 doc_targets += acpi.1
 endif
 
-PREFIX := /usr/local
+PREFIX := /usr
 
 all: $(targets)
 
